File: String/1935. Maximum Number of Words You Can Type.py
class Solution:
    def canBeTypedWords(self, text: str, brokenLetters: str) -> int:
        words = text.split(' ')
        brokenLetters = set(brokenLetters)

        ans = len(words)
        for word in words:
            word = set(word)
            for ch in word:
                if ch in brokenLetters:
                    ans -= 1
                    break
        return ans

File: String/1935. Maximum Number of Words You Can Type.cpp
class Solution {
public:
    int canBeTypedWords(string text, string brokenLetters) 
    {
        string s;
        int ans = 0;
        stringstream ss(text);
        while(ss >> s)
        {
            bool flag = true;
            for(char c : s)
            {
                if(brokenLetters.find(c) != -1)
                {
                    flag = false;
                    break;
                }
            }
            ans += flag;
        }
        return ans;
    }
};